app/routes.py

```diff
@@ -1,2 +1,172 @@
 from flask import Blueprint
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, make_response, jsonify, request
+from app import db
+from sqlalchemy import asc, desc
+from datetime import date
+import requests
+import os
 
+
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+def return_404():
```

Comment:
The name `return_404` is misleading since you still need to return the value produced by this function.  

I suggest naming things like this something like `make_404` in the future:

```suggestion
def make_404():
```